firebase-ai/src/test/java/com/google/firebase/ai/util/tests.kt

```diff
@@ -212,7 +233,22 @@ internal fun goldenVertexUnaryFile(
   name: String,
   httpStatusCode: HttpStatusCode = HttpStatusCode.OK,
   block: CommonTest,
-) = goldenUnaryFile("vertexai/$name", httpStatusCode, block)
+) = goldenUnaryFile("vertexai/$name", httpStatusCode, block = block)
+
+/**
+ * A variant of [goldenUnaryFile] for developer api tests Loads the *Golden File* and automatically
+ * provides it to the channel.
+ *
```

Comment:
```suggestion
/**
 * A variant of [goldenUnaryFile] for developer api tests.
 * 
 * Loads the *Golden File* and automatically provides it to the channel.
 *
```

Reply:
I'm not sure I understand the diff here.